code/modules/jobs/job_types/bos.dm

```diff
@@ -979,6 +983,8 @@ Initiate
 
 	outfit = /datum/outfit/job/bos/f13initiate
 
+	access = list(ACCESS_BOS)
```

Comment:
Ditto.

Reply:
To solve this issue, I just made sure the paths that BoS has access to are updated to be for ACCESS_BOS and not isolated to a ss13 department. This will lay the foundation for the access code rework to be catered to f13 factions, rather than rehashing ss13 setups.